schranz-search/symfony-bundle/0.2/manifest.json

```diff
@@ -0,0 +1,26 @@
+{
+    "bundles": {
+        "Schranz\\Search\\Integration\\Symfony\\SearchBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "env": {
+        "#1": "Install one of the following adapters: https://schranz-search.github.io/schranz-search/getting-started/index.html",
+        "#2": "SEAL_DSN=\"meilisearch://127.0.0.1:7700\"",
+        "#3": "SEAL_DSN=\"algolia://ALGOLIA_APPLICATION_ID:ALGOLIA_ADMIN_API_KEY\"",
+        "#4": "SEAL_DSN=\"elasticsearch://127.0.0.1:9200\"",
+        "#5": "SEAL_DSN=\"opensearch://127.0.0.1:9200\"",
+        "#6": "SEAL_DSN=\"redisearch://127.0.0.1:6379\"",
+        "#7": "SEAL_DSN=\"solr://127.0.0.1:8983\"",
+        "#8": "SEAL_DSN=\"typesense://S3CR3T@127.0.0.1:8108\"",
+        "SEAL_DSN": "loupe://var/indexes"
+    },
+    "add-lines": [
+        {
+            "file": ".env.test",
+            "content": "SEAL_DSN=memory://",
+            "position": "bottom"
+        }
+    ]
```

Comment:
@dhirtzbruch got some good suggestion to move this to the .yaml config there we could also then define the index_name_prefix: https://github.com/schranz-search/schranz-search/issues/316#issuecomment-1857838457